tests/HtmlMatcherTest.php

```diff
@@ -78,14 +78,12 @@ public function considersValidHtml_WHtmlContainsScriptTagWithHtmlContents()
     public function addsSpecificTextInsideTheSciptTagsInsteadOfItsContents()
     {
         $html = "<div>
-<script type='x-template'>
-	<span></span>
-</script>
+<script type='x-template'><span></span></script>
 </div>";
 
         assertThat($html, is(htmlPiece(havingChild(
             both(withTagName('script'))
-                ->andAlso(havingTextContents(HtmlMatcher::SCRIPT_BODY_REPLACEMENT))))));
+                ->andAlso(havingTextContents("<span><\\/span>"))))));
```

Comment:
Everything you wrote sounds like you are afraid of vertical space. Oh dear. Can you try to disable a few toolbars and pinned windows and use the gained space to create code that exposes it's structure a little more obviously? I notice for a while now that you tend to cram multiple things into one line. I, personally, try to not do this any more since I am not coding on 25 rows text mode screens any more.